fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/drawer/Drawer.kt

```diff
@@ -255,15 +255,16 @@ fun rememberDrawerState(confirmValueChange: (DrawerValue) -> Boolean = { true })
 
```

Comment:
don't we need changes for Drawer as well?

Reply:
Updated to Drawer as well